app/src/main/java/org/schabi/newpipe/error/ErrorPanelHelper.kt

```diff
@@ -99,6 +103,7 @@ class ErrorPanelHelper(
             }
 
             errorRetryButton.isVisible = true
+            errorOpenInBrowserButton.isVisible = true
```

Comment:
Shouldn't the on click listener be set here, too? I suggest that you create a `showAndSetOpenInBrowserButtonAction` function which sets the button to visible and sets the on click listener. Then you can call that function both here and 30 lines below here ;-)